test/scenarios/mixed-reality/input/stable/2020-05-01/test-scenarios/depSto.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+  },
+  "variables": {
+    "storageAccountName": "[uniqueString(resourceGroup().id)]"
+  },
+  "resources": [{
+    "type": "Microsoft.Storage/storageAccounts",
+    "apiVersion": "2019-06-01",
```

Comment:
How do we or customers whoever should write this arm template know which api versions for depend RP resources they should use? what if this RP has used some advanced features of higher api-versions in  Microsoft.Storage/storageAccounts ?

Reply:
It's something the service side should tell us. From codegen point of view, we can just trust the arm-template content.